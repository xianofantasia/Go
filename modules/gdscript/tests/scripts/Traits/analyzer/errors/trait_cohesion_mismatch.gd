uses SomeTrait, OtherTrait

trait SomeTrait:
		func func_one():
				print("function one")

trait OtherTrait:
		func func_one():
				print("function two")

func test():
		print("ok")
