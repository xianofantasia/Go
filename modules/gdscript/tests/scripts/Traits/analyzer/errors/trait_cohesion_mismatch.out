GDTEST_ANALYZER_ERROR
>> ERROR at line 1: Can not uses "OtherTrait" trait since shadows members from previous traits.
